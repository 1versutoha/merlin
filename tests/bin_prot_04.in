["find", "use", "bin_prot"]
["tell", "struct", "module Test_binprot : sig type 'a t = { x : int ; y : 'a } with bin_io end = struct type 'a t = { x : int ; y : 'a } with bin_io end"]
["tell", "struct", null]
["dump", "env"]
