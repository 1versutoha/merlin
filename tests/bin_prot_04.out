["return",true]
["return",false]
["return",true]
["return",[["Line 1, characters 31-59","module Test_binprot :\n  sig\n    type 'a t = { x : int; y : 'a; }\n    val bin_size_t : ('a -> int) -> 'a t -> int\n    val bin_write_t :\n      (Bin_prot.Unsafe_common.sptr ->\n       Bin_prot.Unsafe_common.eptr -> 'a -> Bin_prot.Unsafe_common.sptr) ->\n      Bin_prot.Common.buf ->\n      pos:Bin_prot.Common.pos -> 'a t -> Bin_prot.Common.pos\n    val bin_writer_t :\n      'a Bin_prot.Type_class.writer -> 'a t Bin_prot.Type_class.writer\n    val bin_read_t :\n      (Bin_prot.Unsafe_common.sptr_ptr -> Bin_prot.Unsafe_common.eptr -> 'a) ->\n      Bin_prot.Common.buf -> pos_ref:Bin_prot.Common.pos ref -> 'a t\n    val bin_reader_t :\n      'a Bin_prot.Type_class.reader -> 'a t Bin_prot.Type_class.reader\n  end"]]]
